cmd/cli/config/default.go

```diff
@@ -1,28 +1,28 @@
 package config
 
 import (
-	"github.com/bacalhau-project/bacalhau/pkg/config"
 	"github.com/spf13/cobra"
-	"github.com/spf13/viper"
 	"gopkg.in/yaml.v3"
+
+	"github.com/bacalhau-project/bacalhau/pkg/config"
 )
 
-func newDefaultCmd() *cobra.Command {
+func newDefaultCmd(cfg config.Context) *cobra.Command {
 	showCmd := &cobra.Command{
 		Use:   "default",
 		Short: "Show the default bacalhau config.",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return defaultConfig(cmd)
 		},
 	}
-	showCmd.PersistentFlags().String("path", viper.GetString("repo"), "sets path dependent config fields")
+	showCmd.PersistentFlags().String("path", cfg.System().GetString("repo"), "sets path dependent config fields")
 	return showCmd
 }
```

Comment:
We could just delete this command in favor of clearer documentation around what the default config structure is since lack of said docs are why it existed in the first place.